router-bridge/src/worker.rs

```diff
@@ -302,6 +303,30 @@ fn op_crypto_get_random_values(state: &mut OpState, out: &mut [u8]) -> Result<()
     Ok(())
 }
 
+// HeapStats stores values from a isolate.get_heap_statistics() call
+#[derive(Serialize)]
+#[serde(rename_all = "camelCase")]
+struct MemoryUsage {
+    //rss: usize,
```

Comment:
keep? remove?

Reply:
I can make a follow up PR to remove that, but fixing right in the release PR is probably a bad idea

Reply:
commented below out of thread. understood. ship it.